python/mxnet/symbol.py

```diff
@@ -1025,6 +1025,36 @@ def grad(self, wrt):
         return Symbol(handle)
     # pylint: enable= no-member
 
+    def eval(self, args=None, ctx=cpu()):
+        if args == None:
```

Comment:
Originally, I wanted to make this (self, args={}, ...)

Mu vetoed this on account of nonconformity with established Python guidelines (prohibiting making an empty dictionary a default argument). 

If you have a better fix that doesn't break style guidelines let me know, otherwise perhaps we'll have to settle for this.